applications/SwimmingDEMApplication/python_scripts/swimming_DEM_analysis.py

```diff
@@ -33,12 +33,12 @@ def Say(*args):
     from KratosMultiphysics.mpi import *
```

Comment:
this is deprecated
use the MPI-Core / dta-communnicator instead, see the [tests](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/mpi/tests/test_mpi_data_communicator_python.py)